dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallServerTest.java

```diff
@@ -155,6 +156,28 @@ public void testInvalidMultiVersionFeatures() throws Exception {
 	checkCommandOutput(po, 21, "CWWKF1405E", null);
 	Log.exiting(c, METHOD_NAME);
     }
+    
+    /**
+     * Test the install of versionless servlet from maven central. Multi-version is not
+     * supported with installServerFeature as it cannot be installed to same
+     * resource.
+     *
+     * @throws Exception
+     */
+    @Ignore
+    @Test
+    public void testVersionlessWithPlatformFeatures() throws Exception {
+	final String METHOD_NAME = "testInvalidMultiVersionFeatures";
+	Log.entering(c, METHOD_NAME);
+
+	copyFileToMinifiedRoot("usr/servers/serverX", "publish/tmp/versionlessWPlatform/server.xml");
+
+	String[] param1s = { "installServerFeatures", "serverX", "--verbose" };
+	ProgramOutput po = runFeatureUtility(METHOD_NAME, param1s);
+
+	checkCommandOutput(po, 21, "CWWKF1405E", null);
```

Comment:
```suggestion
	String[] filesList = { "/lib/features/com.ibm.websphere.appserver.servlet-6.0.mf" };
	checkCommandOutput(po, 0, null, filesList);
```

Reply:
Thanks - these tests will be enabled and fixed after the repo PR is finished...  probably a few days from now...